.vscode/tasks.json

```diff
@@ -0,0 +1,21 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "npm",
+			"script": "watch",
+			"isBackground": true,
+			"problemMatcher": "$ts-esbuild-watch",
+            "options": {
+                "cwd": "${workspaceFolder}/editors/vscode"
+            },
```

Comment:
it seems there are mixed spaces and tabs

Reply:
Good catch, thanks